tests/benchmarks/tdigits10.nim

```diff
@@ -0,0 +1,88 @@
+#[
+benchmark for digits10
+]#
+
+func digits10v1(num: uint64): int {.noinline.} =
```

Comment:
Did you try removing noinline pragma?

Reply:
yes, see last commit + updated benchmark

Reply:
Thanks